code/defines/procs/records.dm

```diff
@@ -7,7 +7,7 @@
 	general_record.fields["real_rank"] = "Unassigned"
 	general_record.fields["sex"] = "Male"
 	general_record.fields["age"] = "Unknown"
-	general_record.fields["ethnicity"] = "Unknown"
+	general_record.fields["skin_color"] = "Unknown"
```

Comment:
💀  i think ethnicity sounds better in this section

Reply:
I mean kinda but it would say “ethnicity = tan 2” (does anyone even look at these records?)